### ISSUE TITLE
May you please provide the train.py?

### ISSUE THREAD

**Issue by xujinfan** [NivNayman/XNAS#1]
Thanks for your codes. May you provide the train.py?  We still don't know how you implement it.

**Comment by debadeepta** [comment]
Indeed! Would be very interested in the train_search scripts as well. That will make the entire paper reproducible.

**Comment by leeyeehoo** [comment]
I think it's hard to reproduce their claimed results. Based on their paper they use 1500 epochs to train... on Cifar10. It's definitely an unfair comparison between his and others works.

**Comment by Drxdx** [comment]
hello Niv Nayman:
I'm very sorry to bother you. 
Recently, I read your XNAS article and very interested in it.And I want to retrain the personalized model according to the XNAS algorithm, I take the liberty of asking you: I wonder if you can share the training code, thank you !  Daxuan
